notebooks/data-sources/TestGrid/testgrid_metadata_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
      "name": "stdout",
```

Comment:
<p>It looks like its now zero not 9</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>